test/e2e/case5_policy_automation_test.go

```diff
@@ -140,6 +140,17 @@ var _ = Describe("Test policy automation", func() {
 
 				return len(ansiblejobList.Items)
 			}, 30, 1).Should(Equal(1))
+			By("Job TTL should match default (1 day)")
+			Eventually(func() interface{} {
+				ansiblejobList, err := clientHubDynamic.Resource(gvrAnsibleJob).Namespace(testNamespace).List(
+					context.TODO(), metav1.ListOptions{},
+				)
+				Expect(err).To(BeNil())
+				_, err = utils.KubectlWithOutput("get", "ansiblejobs", "-n", testNamespace)
+				Expect(err).Should(BeNil())
+
+				return ansiblejobList.Items[0].Object["spec"].(map[string]interface{})["job_ttl"]
+			}, 10, 1).Should(Equal(int64(86400)))
```

Comment:
This is so that the `Eventually` won't stop trying if an assertion fails.
```suggestion
			Eventually(func(g Gomega) interface{} {
				ansiblejobList, err := clientHubDynamic.Resource(gvrAnsibleJob).Namespace(testNamespace).List(
					context.TODO(), metav1.ListOptions{},
				)
				g.Expect(err).To(BeNil())
				_, err = utils.KubectlWithOutput("get", "ansiblejobs", "-n", testNamespace)
				g.Expect(err).Should(BeNil())

				return ansiblejobList.Items[0].Object["spec"].(map[string]interface{})["job_ttl"]
			}, 10, 1).Should(Equal(int64(86400)))
```